starknet-core/src/serde/u64.rs

```diff
@@ -0,0 +1,14 @@
+pub mod fee_amount {
```

Comment:
This file can be now deleted as well, right?

Reply:
Wait how come I didn't delete it lol